diff --git a/tools/staticlib.R b/tools/staticlib.R
index 84f841a..33b088c 100644
--- a/tools/staticlib.R
+++ b/tools/staticlib.R
@@ -1,6 +1,7 @@
 target_args <- if ( .Platform$OS.type == "windows" ) c("--target", cargo::target()) else NULL
 
-if ( cargo::run("build", target_args, "--release", "--manifest-path", "rustlib/Cargo.toml", minimum_version="1.49") ) {
+Sys.setenv(R_CARGO_SAVE_CACHE="TRUE") # to not reset cargo_home to something in tmp
+if ( cargo::run("build", target_args, "--release", "--offline", "--manifest-path", "rustlib/Cargo.toml", minimum_version="1.49") ) {
 
   if ( ! is.null(target_args) ) {
     args <- commandArgs(TRUE)
@@ -11,6 +12,7 @@ if ( cargo::run("build", target_args, "--release", "--manifest-path", "rustlib/C
   }
 
 } else {
+print(c("cargo_home", Sys.getenv("CARGO_HOME")))
 
   cargo:::download_staticlib(
     "https://r.ddahl.org/staticlib/${name}_${version}/${target}.tar.gz"
