diff --git a/R/aa-global.R b/R/aa-global.R
index dbc32f7..c24e877 100644
--- a/R/aa-global.R
+++ b/R/aa-global.R
@@ -28,23 +28,28 @@ prepare_KEGG <- get("prepare_KEGG",
 
 #' @import ExperimentHub
 eh <- tryCatch(suppressMessages(ExperimentHub()), error=function(e){
-    refreshHub(hubClass="ExperimentHub")
+    tryCatch(refreshHub(hubClass="ExperimentHub"), error=function(e) {
+	print("ExperimentHub not ok, and refreshHub call failed")
+	F
+	})
+   F
 })
+if (eh == F) {
+} else {
+	#' @importFrom BiocGenerics fileName
+	validLoad <- function(ehid){
+	    tryCatch(suppressMessages(eh[[ehid]]), 
+		     error=function(e){
+			 unlink(fileName(eh[ehid]))
+			 eh[[ehid]]})
+	}
 
-#' @importFrom BiocGenerics fileName
-validLoad <- function(ehid){
-    tryCatch(suppressMessages(eh[[ehid]]), 
-             error=function(e){
-                 unlink(fileName(eh[ehid]))
-                 eh[[ehid]]})
+	# GO_DATA <- get_GO_data(OrgDb, ont, keytype="SYMBOL")
+	# download GO_DATA.rds from AnnotationHub to save time by avoiding 
+	# building GO_DATA from scratch
+	GO_DATA <- validLoad("EH3231")
+	# GO_DATA_drug <- get_GO_data_drug(OrgDb = "org.Hs.eg.db", 
+	#                                  ont, keytype="SYMBOL")
+	# download GO_DATA_drug.rds 
+	GO_DATA_drug <- validLoad("EH3232")
 }
-
-# GO_DATA <- get_GO_data(OrgDb, ont, keytype="SYMBOL")
-# download GO_DATA.rds from AnnotationHub to save time by avoiding 
-# building GO_DATA from scratch
-GO_DATA <- validLoad("EH3231")
-# GO_DATA_drug <- get_GO_data_drug(OrgDb = "org.Hs.eg.db", 
-#                                  ont, keytype="SYMBOL")
-# download GO_DATA_drug.rds 
-GO_DATA_drug <- validLoad("EH3232")
-
